### ISSUE TITLE
Problem when evaluating CenterPoint using Waymo's official evaluation tool

### ISSUE THREAD

**Issue by yuhuan-wu** [tianweiy/CenterPoint#158]
Dear Tianwei and other Contributors,

Your CenterPoint is very Inspiring! Congratulations that you have achieved the best performance maintaining a high speed.

I am reproducing CenterPoint on Waymo dataset. I use the newest version of this repository. 

At the first time I train CenterPoint with config  `configs/waymo/voxelnet/waymo_centerpoint_voxelnet_6epoch.py`, and then I use your test script `tools/dist_test.py` to generate `detection_pred.bin` for the evaluation.

Also following your suggestion, I generate GT detections `gt_pred.bin` using the command which is same as `docs/WAYMO.md`:
```
python det3d/datasets/waymo/waymo_common.py --info_path data/Waymo/infos_val_01sweeps_filter_zero_gt.pkl --result_path data/Waymo/ --gt 
```

Following Waymo dataset's instructions, I installed `bazel` and use the following command to do the evaluation: 

```
bazel-bin/waymo_open_dataset/metrics/tools/compute_detection_metrics_main detection_pred.bin gt_pred.bin
```

However, after several seconds, an error occurred:

```
F0712 09:02:58.438066  5649 metrics_utils.cc:85] Check failed: pd_heading >= -3.14159265358979323846 - kError (-3.16124 vs. -3.14159) 
```

It seems that the predicted heading is out of [-pi, pi]. 

I wonder whether this is my fault so I also tried evaluating your released prediction binary files (`waymo_centerpoint_voxelnet_3x/detection_pred.bin` of your google drive shared folder). **However, same error occurred.**

I think a straightforward solution is to convert the out-ranged predicted heading to [-pi, pi].

Since I am new to 3D tasks, based on my findings, I have several questions:

1. How to solve the above error? How to  convert the out-ranged predicted heading to [-pi, pi]? Will such conversion affect the performance?
2. Why is this error occurred? Does there exist some bugs in this repository? Could we solve it during the testing stage?
3. I found no others mentioned such issue. So I am also guessing whether my data are broken. However, I have checked all the gt heading values and no heading values are out of ranging [-pi, pi].

**Comment by tianweiy** [comment]
Hi, which version of waymo devkit are you using ? Could you try the latest one ? Also, you can submit the detection result to the server for evaluation. I never met this issue in the past

**Comment by yuhuan-wu** [comment]
> Hi, which version of waymo devkit are you using ? Could you try the latest one ? Also, you can submit the detection result to the server for evaluation. I never met this issue in the past

Hi, tianwei! After carefully checking the version of each op, I find that I cloned `v1.0-tf1.15.0 waymo-od toolkit` from the waymo's official repo, which is inconsistent with the version of what Centerpoint used (v1.2). 
So I have addressed the reported issue by cloning a `v1.2 waymo-od  toolkit`. 

Thanks for your help!
